doc/rados/configuration/pool-pg-config-ref.rst

```diff
@@ -164,3 +172,4 @@ Ceph configuration file.
 
 .. _pool: ../../operations/pools
 .. _Monitoring OSDs and PGs: ../../operations/monitoring-osd-pg#peering
+.. _Weighting Bucket Items: ../../operations/crush-map#Weighting Bucket Items
```

Comment:
again this does not work. "Weighting Bucket Items" does not point to any anchor in crush-map.rst. you might want to create a hyper link target in that document first, then reference it from here.

i'd suggest you try `./admin/build-doc` and test before submitting the document related change.